src/main/resources/application.properties

```diff
@@ -1,2 +1,7 @@
 spring.profiles.active=dev
-spring.mvc.hiddenmethod.filter.enabled=true
\ No newline at end of file
+spring.mvc.hiddenmethod.filter.enabled=true
+spring.mail.host=smtp.centrum.cz
+spring.mail.port=587
+spring.config.import = classpath:/email.properties

```

Comment:
This is a nice solution but it would be easier to use environment variables. We could fill in some placeholders like spring.mail.password=changethis and then overwrite it using an environment variable. I will show you how to do it after the scrum meeting, we can discuss which option to use